### PR TITLE
Update RaspberryPi instructions

### PR DESCRIPTION
**Pull request by MAKIO135**
- update Node to 12.x
- add install command
- add required permissions for `/usr/lib/node_modules`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,9 @@ Connecting to EBB on /dev/tty.usbmodem1461
 To install saxi on a Raspberry Pi, first install node.js if you haven't already:
 
 ```
-$ curl -sL https://deb.nodesource.com/setup_11.x | sudo -E bash -
+$ curl -sL https://deb.nodesource.com/setup_12.x | sudo -E bash -
+$ sudo apt-get install -y nodejs
+$ sudo chmod a+rwx /usr/lib/node_modules
```
Comment (nornagon) [inline README.md:32]:
```suggestion
```
I don't think this is advisable. Here's what npm's documentation has to say about installing modules globally without root: https://docs.npmjs.com/resolving-eacces-permissions-errors-when-installing-packages-globally

**Comment by nornagon** [comment]
Thanks for the PR! I removed the chmod instructions, but I didn't yet add in any other information about how to resolve "permission denied" issues when running `npm install -g`. I'd be happy to review another PR if you're keen to add that back in with a link to npm's instructions instead!

File: README.md (hunk 1 above)
Comment (MAKIO135) [reply README.md]: Haha, yes, I went with this because it was for raspberrypi, wouldn't do that on my computer 🙃

**Comment by MAKIO135** [comment]
🙏I'll try a new PR soon

**Pull request merged**